### PR TITLE
Ethereum address creation faulty

### PR DESCRIPTION
**Pull request by nolash**
Hashing of ethereum address by public key was not called correctly.

See https://github.com/ethersphere/bee/pull/345/files#diff-aa4c96b4d3b1b4d4fcd8f914144cd943L59

### CODE REVIEW

File: pkg/crypto/crypto_test.go [hunk 1]
```diff
@@ -37,7 +38,10 @@ func TestNewAddress(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	a := crypto.NewOverlayAddress(k.PublicKey, 1)
+	a, err := crypto.NewOverlayAddress(k.PublicKey, 1)
```
Comment (acud) [inline pkg/crypto/crypto_test.go:41]: how is it that here `NewOverlayAddress` is called with `1` and in the test above it is `3`?

Comment (nolash) [reply pkg/crypto/crypto_test.go:41]: no idea, I just left is as it was

Comment (acud) [reply pkg/crypto/crypto_test.go:41]: k

Comment (pradovic) [reply pkg/crypto/crypto_test.go:41]: @acud  That is a networkID argument. If you think the tests in bzz/address_test.go by test above, I think I just used number 3 randomly :)

**Pull request merged**